tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AdoNet/ReaderReadAsyncIntegration.cs

```diff
@@ -0,0 +1,71 @@
+// <copyright file="ReaderReadAsyncIntegration.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+using System;
+using System.ComponentModel;
+using System.Threading;
+using Datadog.Trace.ClrProfiler.CallTarget;
+using Datadog.Trace.Iast;
+using Datadog.Trace.Vendors.Serilog;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.AdoNet
+{
+    /// <summary>
+    /// CallTarget instrumentation for:
+    /// [*]DataReader [Command].ExecuteReader()
+    /// </summary>
+    [Browsable(false)]
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class ReaderReadAsyncIntegration
+    {
+        private static bool errorLogged = false;
+
+        /// <summary>
+        /// OnMethodBegin callback
+        /// </summary>
+        /// <typeparam name="TTarget">Type of the target</typeparam>
+        /// <param name="instance">Instance value, aka `this` of the instrumented method.</param>
+        /// <param name="cancellationToken">CancellationToken value</param>
+        /// <returns>Calltarget state value</returns>
+        internal static CallTargetState OnMethodBegin<TTarget>(TTarget instance, CancellationToken cancellationToken)
+        {
+            return new CallTargetState(null);
```

Comment:
I don't really understand why `null` here (and in the other `OnMethodBegin`)?

Reply:
Because I don't want to generate a span, I just want to intercept the call